src/test/java/teammates/logic/api/EmailGeneratorTest.java

```diff
@@ -217,14 +217,23 @@ public void testGenerateFeedbackSessionEmails() throws Exception {
         subject = String.format(EmailType.FEEDBACK_OPENING_SOON.getSubject(), course.getName(),
                 session.getFeedbackSessionName());
 
-        // this instructor email has been given co-owner privileges in the test file
-        InstructorAttributes coOwner1 =
+        // this instructor email has been given co-owner privileges in the test file but has not joined
+        InstructorAttributes coOwnerNotJoined =
                 instructorsLogic.getInstructorForEmail(course.getId(), "instructorNotYetJoinedCourse1@email.tmt");
 
-        assertTrue(coOwner1.hasCoownerPrivileges());
+        assertTrue(coOwnerNotJoined.hasCoownerPrivileges());
 
-        verifyEmailReceivedCorrectly(emails, coOwner1.getEmail(), subject,
-                "/sessionOpeningSoonEmailForCoOwner.html");
+        verifyEmailReceivedCorrectly(emails, coOwnerNotJoined.getEmail(), subject,
+                "/sessionOpeningSoonEmailForCoOwnerNotJoined.html");
+
+        // this instructor email has been given co-owner privileges in the test file but has not joined
```

Comment:
This instructor has joined, though?

Reply:
My bad, meant for this to say the instructor has joined. Made the modifications.